guides/common/modules/proc_creating-a-custom-file-type-repository.adoc

```diff
@@ -17,7 +17,7 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-custom
 . Optional: In the *Upstream URL* field, enter the URL of the upstream repository to use as a source.
 If you do not enter an upstream URL, you can manually upload packages.
 For more information, see xref:Uploading_Files_To_a_Custom_File_Type_Repository_{context}[].
-. Select the *Verify SSL* checkbox if you want to verify that the upstream repository's SSL certificates are signed by a trusted CA.
+. Select the *Verify SSL* checkbox if you want to verify that the SSL certificates of the repository are signed by a trusted CA.
```

Comment:
Small nit - Normally we do not specify the UI element if there is no chance of ambiguity. Also, I think you can remove "if you want to" because we assume that the user wants to do the action. 

I recommend you change this to `Select *Verify SSL* to verify that the SSL certificates of the repository are signed by a trusted CA.`